code/datums/brain_damage/imaginary_friend.dm

```diff
@@ -77,7 +77,7 @@
 	var/hidden = FALSE
 	var/move_delay = 0
 	var/mob/living/owner
-	var/bubble_icon = "default"
+	// MONKESTATION REMOVAL var/bubble_icon = "default"
```

Comment:
```suggestion
	//var/bubble_icon = "default" // MONKESTATION REMOVAL
```
more consistent with how we usually do it, it separates the "removal" tag from what we removed